packages/preferences/src/browser/views/preference-searchbar-widget.tsx

```diff
@@ -53,20 +55,87 @@ export class PreferencesSearchbarWidget extends ReactWidget {
         }
     }
 
+    /**
+     * Clears the search input and all search results.
+     * @param e on-click mouse event.
+     */
+    protected clearSearchResults = (e: React.MouseEvent): void => {
+        const search = document.getElementById(PreferencesSearchbarWidget.SEARCHBAR_ID) as HTMLInputElement;
+        if (search) {
+            search.value = '';
+            this.search(search.value);
+            this.update();
+        }
+    };
+
+    /**
+     * Renders all search bar options.
+     */
+    protected renderOptionContainer(): React.ReactNode {
+        const resultsCount = this.renderResultsCountOption();
+        const clearAllOption = this.renderClearAllOption();
+        return <div className="option-buttons"> {resultsCount} {clearAllOption} </div>;
+    }
+
+    /**
+     * Renders a badge displaying search results count.
+     */
+    protected renderResultsCountOption(): React.ReactNode {
+        const resultsFound = `${this.resultsCount === 0 ? 'No' : this.resultsCount} ${this.resultsCount === 1 ? 'Setting Found' : 'Settings Found'}`;
+        return this.searchTermExists() ?
+            (<span
+                className="results-found"
+                title={resultsFound}>
+                {resultsFound}
+            </span>)
+            : '';
+    }
+
+    /**
+     * Renders a clear all button.
+     */
+    protected renderClearAllOption(): React.ReactNode {
+        return <span
+            className={`clear-all option ${(this.searchTermExists() ? 'enabled' : '')}`}
+            title="Clear Search Results"
+            onClick={e => this.clearSearchResults(e)}
```

Comment:
Since you've defined `clearSearchResults` as a property above (arrow function rather than method), you should be able to rewrite this as `onClick={this.clearSearchResults}` and avoid putting an arrow function in a React component.